### ISSUE TITLE
UTF-8 required for automatic reloading

### ISSUE THREAD

**Issue by akirataguchi115** [yandeu/five-server#142]
**Describe the bug**
I had to add `<head><meta charset="UTF-8"></head>` for the five-server to start automatically reloading. Courtesy of: https://stackoverflow.com/questions/70249584/live-server-not-refreshing-browser . What's the explanation to this? Not sure if it is a bug but it is not self-explanatory either.

**Comment by yandeu** [comment]
Five Server needs to inject some code into the head. 
But I believe you only need a minimum of an empty html, body and head tag for it to work. So no meta+utf8.

See https://github.com/yandeu/five-server/blob/main/src/middleware/injectCode.ts

---

Doesn't it work with just the code below?

```html
<!DOCTYPE html>
<html>
  <head></head>
  <body></body>
</html>
```

**Comment by akirataguchi115** [comment]
You're right the meta+utf8 was indeed not the problem here. It seems that someone has addressed the injecting "issue" back in 2016: https://stackoverflow.com/a/25749523/7629743

A simple example 
```html
<!doctype html>
<head></head>
```

autoreloads fine but I'm not using the head (nor html or body) element at all in this case.

Justifying this with The Google Style Guide for HTML would be pure insanity from my side but on my case five-server just doesn't work.

Should I leave this issue open for future ideas or should I close this as "not planned"?
